modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts

```diff
@@ -75,24 +75,49 @@ const renderStatusbar = (editor: Editor, providersBackstage: UiFactoryBackstageP
 
   const getTextComponents = (): SimpleSpec[] => {
     const components: SimpleSpec[] = [];
+    const shouldRenderHelp = Options.useHelpAccessibility(editor);
+    const shouldRenderElementPath = Options.useElementPath(editor);
+    const shouldRenderRightContainer = Options.useBranding(editor) || editor.hasPlugin('wordcount');
 
-    if (Options.useElementPath(editor)) {
+    const getTextComponentClass = () => {
```

Comment:
Having this plural would be more accurate
```suggestion
    const getTextComponentClasses = () => {
```